Formula/coccinelle.rb

```diff
@@ -33,11 +33,11 @@ def install
     system "opam", "config", "exec", "--", "make"
     system "make", "install"
 
-    share.install ["demos/simple.cocci", "demos/simple.c"]
+    pkgshare.install ["demos/simple.cocci", "demos/simple.c"]
   end
 
   test do
-    system "#{bin}/spatch", "-sp_file", "#{share}/simple.cocci", "#{share}/simple.c", "-o", "new_simple.c"
+    system "#{bin}/spatch", "-sp_file", "#{pkgshare}/simple.cocci", "#{pkgshare}/simple.c", "-o", "new_simple.c"
```

Comment:
Please wrap split this into two lines so it doesn't go passed the eightieth column like this:
```
    system "#{bin}/spatch", "-sp_file", "#{pkgshare}/simple.cocci",
                            "#{pkgshare}/simple.c", "-o", "new_simple.c"
```